styles/globals.scss

```diff
@@ -1124,3 +1124,9 @@ html:has(dialog[data-cy="formLinkerModal"][open]) {
 .action-card-info-icon:hover {
   transform: scale(1.1);
 }
+
+.tooltipContent {
+  max-width: 500px;
```

Comment:
Hi @DorisWongNHS , how does this width look on mobile?

Reply:
![image](https://github.com/user-attachments/assets/224d4582-4455-40cc-a3f1-3ef1af875904)
I have keep it as 500px in wide screen, for screen smaller than 500px width, it will show depends on the parent div width.